db/blob/db_blob_basic_test.cc

```diff
@@ -367,6 +367,37 @@ TEST_F(DBBlobBasicTest, MultiGetMergeBlobWithPut) {
   ASSERT_EQ(values[2], "v2_0");
 }
 
+#ifndef ROCKSDB_LITE
+TEST_F(DBBlobBasicTest, BlobDBProperties) {
+  Options options = GetDefaultOptions();
+  options.enable_blob_files = true;
+  options.min_blob_size = 0;
+  Reopen(options);
+
+  constexpr size_t num_files = 10;
```

Comment:
This is just a suggestion but you could consider using a lower number (e.g. 2) here to make the test minimal and potentially make the validation we discussed simpler.